extras/dbx_to_snowflake/source_to_jupyter.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+from constants import *
+from common import *
```

Comment:
Import functions explicitly to improve readability
```suggestion
from common import create_notebook, save_notebook
```